src/queues/mock.rs

```diff
@@ -0,0 +1,136 @@
+                        email_address: String::from(
+                            "fxa-email-service.queues.mock.bounce@example.com",
+                        ),
+                        action: None,
+                        status: None,
+                        diagnostic_code: None,
+                    }],
+                    timestamp: Utc::now(),
+                    feedback_id: String::from(""),
+                    remote_mta_ip: None,
+                    reporting_mta: None,
+                });
+                bounce_message
+            }
+
+            "incoming-complaint" => {
+                let mut complaint_message = Message::default();
+                complaint_message.notification.notification_type = NotificationType::Complaint;
+                complaint_message.notification.complaint = Some(Complaint {
+                    complained_recipients: vec![ComplainedRecipient {
+                        email_address: String::from(
+                            "fxa-email-service.queues.mock.complaint@example.com",
+                        ),
+                    }],
+                    timestamp: Utc::now(),
+                    feedback_id: String::from(""),
+                    user_agent: None,
+                    complaint_feedback_type: None,
+                    arrival_date: Utc::now(),
+                });
+                complaint_message
+            }
+
+            "incoming-delivery" => {
+                let mut delivery_message = Message::default();
+                delivery_message.notification.notification_type = NotificationType::Delivery;
+                delivery_message.notification.delivery = Some(Delivery {
+                    timestamp: Utc::now(),
+                    processing_time_millis: 0,
+                    recipients: vec![String::from(
+                        "fxa-email-service.queues.mock.delivery@example.com",
+                    )],
+                    smtp_response: String::from(""),
+                    remote_mta_ip: None,
+                    reporting_mta: None,
+                });
+                delivery_message
+            }
+
+            "incoming-bounce-error" => {
+                let mut insane_message = Message::default();
+                insane_message.notification.notification_type = NotificationType::Bounce;
+                insane_message.notification.complaint = Some(Complaint {
+                    complained_recipients: vec![ComplainedRecipient {
+                        email_address: String::from(
+                            "fxa-email-service.queues.mock.complaint@example.com",
+                        ),
+                    }],
+                    timestamp: Utc::now(),
+                    feedback_id: String::from(""),
+                    user_agent: None,
+                    complaint_feedback_type: None,
+                    arrival_date: Utc::now(),
+                });
+                insane_message
```

Comment:
nit: something about the use of "insane" throughout this doesn't sit right with me, can we use a less emotional term like "unexpected"?